protocols/gossipsub/src/handler.rs

```diff
@@ -381,25 +327,16 @@ impl ConnectionHandler for Handler {
                             return Poll::Ready(ConnectionHandlerEvent::Custom(message));
                         }
                         Poll::Ready(Some(Err(error))) => {
-                            match error {
-                                HandlerError::MaxTransmissionSize => {
-                                    warn!("Message exceeded the maximum transmission size");
-                                    self.inbound_substream =
-                                        Some(InboundSubstreamState::WaitingInput(substream));
-                                }
-                                _ => {
-                                    warn!("Inbound stream error: {}", error);
-                                    // More serious errors, close this side of the stream. If the
-                                    // peer is still around, they will re-establish their
-                                    // connection
-                                    self.inbound_substream =
-                                        Some(InboundSubstreamState::Closing(substream));
-                                }
-                            }
+                            log::debug!("Failed to read from inbound stream: {error}");
+                            // Close this side of the stream. If the
+                            // peer is still around, they will re-establish their
+                            // connection
```

Comment:
```suggestion
                            // peer is still around, they will re-establish their
                            // outbound stream i.e. our inbound stream.
```

It is not a connection that they reestablish, but a stream, right?